dir/path.go

```diff
@@ -79,21 +79,35 @@ const (
 
 var userConfigDir = os.UserConfigDir // for unit test
 
-func init() {
-	loadUserPath()
-}
-
 // loadUserPath function defines UserConfigDir and UserLibexecDir.
 func loadUserPath() {
-	// set user config
-	userDir, err := userConfigDir()
-	if err != nil {
-		panic(err)
+	UserConfigDir = ""
+	UserLibexecDir = ""
+
+	_ = UserLibexecDirPath()
```

Comment:
We may don't need to initialize it as it is lazy now.

Reply:
Agreed. I left it in for the sake of the unit tests but I've since removed it. Resolved in [46c35d4](https://github.com/notaryproject/notation-go/pull/449/commits/46c35d45a2c830ad16861f7ab00b6a9ccb8aec9d).